RFS/src/main/java/org/opensearch/migrations/bulkload/transformers/TransformFunctions.java

```diff
@@ -24,7 +24,7 @@ public static Transformer getTransformer(
         MetadataTransformerParams metadataTransformerParams
     ) {
         if (VersionMatchers.isOS_2_X.or(VersionMatchers.isOS_1_X).test(targetVersion)) {
-            if (VersionMatchers.isES_6_X.test(sourceVersion)) {
+            if (VersionMatchers.isES_6_X.or(VersionMatchers.isES_5_X).test(sourceVersion)) {
                 return new Transformer_ES_6_8_to_OS_2_11(dimensionality, metadataTransformerParams);
```

Comment:
At some point we should grapple with our naming strategy here--does `Transformer_ES_6_8_to...` mean `exactly 6.8`? or `up to 6.8` or `same major version as 6.8`--but I'm okay deferring that discussion.

Reply:
We keep the specific names because there is always the potential that we would need an `ES_6_4` specific version...  If you'd like to go in and change the names that is a small PR I'm happy to review and we can deal with minor version changes when they appear.